docs/TempUG.md

```diff
@@ -1188,7 +1210,7 @@ If your changes to the data file makes its format invalid, TutorMate will discar
 **A**: Copy TutorMate's jar file into the folder you want to use as the home folder in the other computer. Before running the application, copy _addressbook.json_, _personLessonMap.json_ and _schedulelist.json_ files from the home folder in your current computer into the home folder of the other computer. After copying the 3 files, running the application will show the same data. However, do note that the changes made through the application in one computer will not be reflected in the data in another computer.
 
 **Q**: Can I edit the data in the application through the _addressbook.json, personLessonMap.json, schedulelist.json_ files directly? <br>
-**A**: It is possible. However, it is not advisable for you to do so as if the changes made to the data file makes its format invalid, TutorMate will discard all data and start with an empty data file at the next run. Please use the `edit` command to make changes to your data instead.
+**A**: It is possible. However, it is not advisable for you to do so as if the changes made to the data file makes its format invalid, TutorMate will discard all data. Read more about editing the storage files [here](#editing-the-data-file). Please use the `edit` command to make changes to your data instead.
```

Comment:
Probably adding this line after "Please use the `edit` command to make changes to your data instead" flows better? Like 
"Please use the `edit` command to make changes to your data instead. But if you insist on directly modifying the raw data, you could refer to ..."

Reply:
Changed in the recent commit, thank you ! ◡̈